libexec/tfenv-uninstall

```diff
@@ -7,21 +7,14 @@ source ${TFENV_ROOT}/libexec/helpers
 
 declare version_requested version regex
 
-if [ -z "${1}" ]; then
-  version_file="$(tfenv-version-file)"
-  if [ "${version_file}" != "${TFENV_ROOT}/version" ];then
-    version_requested="$(cat ${version_file} || true)"
-  fi
-else
-  version_requested="${1}"
-fi
+get_version_requested "${1}"
 
 if [[ "${version_requested}" =~ ^latest\:.*$ ]]; then
   version="${version_requested%%\:*}"
   regex="${version_requested##*\:}"
 elif [[ "${version_requested}" =~ ^latest$ ]]; then
   version="${version_requested}"
-  regex=""
+  regex="^[0-9]\+\.[0-9]\+\.[0-9]\+$"
```

Comment:
Not sure the value of adding this here other than to restrict possibilities. Regex only exists for the purpose of a user specifying one.